src/lib/components/composites/project-components/ProjectListEntry.svelte

```diff
@@ -27,30 +21,37 @@ Furthermore this component is clickable and navigates to the corresponding proje
 
 Usage:
 ```svelte
-    <ProjectListEntry project={demoProject} members={memberUserSpecArray} stage={3} stageProgress={70} />
+    <ProjectListEntry project={demoProject} members={memberUserSpecArray} statistics={{projectProgress: 0.3}} />
 ```
 -->
 <a
     class="flex flex-col lg:flex-row h-fit w-full gap-2 lg:gap-10 lg:items-center justify-between px-5 py-2
-    border border-container-border-grey rounded-md highlight-on-hover {project.archived
+    border border-container-border-grey rounded-md highlight-on-hover {project.status ===
+    ProjectStatus.ARCHIVED
         ? 'opacity-25'
         : ''}"
     href={`/project/${project.id}/dashboard`}
 >
     <div class="flex flex-col min-w-0 h-fit">
         <h2 class="truncate">{project.name}</h2>
 
-        {#if members.length > 0}
-            <span class="truncate text-hint">{getNames(members)}</span>
+        {#if membersList.members.length > 0}
+            <span class="truncate text-hint"
+                >{getNames(
+                    membersList.members
+                        .map((member) => member.user)
+                        .filter((user) => user !== undefined),
+                )}</span
+            >
```

Comment:
Here, the automatic formatting looks weird again.
```suggestion
            <span class="truncate text-hint">
                {getNames(
                    membersList.members
                        .map((member) => member.user)
                        .filter((user) => user !== undefined),
                )}
            </span>
```

We could possibly destruct the `membersList` object to get the `members` directly.

Reply:
I don't think it is the effort worth.